tests/rptest/tests/partition_movement_test.py

```diff
@@ -147,6 +148,62 @@ def derived_done():
 
         wait_until(derived_done, timeout_sec=30, backoff_sec=1)
 
+    @cluster(num_nodes=3)
+    def test_moving_not_fully_initialized_partition(self):
```

Comment:
how does the test work with builds that have honey badger disabled? I suggested a slight change to the routes in a previous commit, and I'm wondering if that would be the basis for dynamically determining if we'll enable/disable this test?

Reply:
In current situation without failure injector, it would just execute like there were no errors. I have applied your suggestion and I am disabling this test if honey badger isn't enabled.